syncano/models/manager.py

```diff
@@ -274,6 +308,7 @@ def serialize(self, data, model=None):
         return model(**properties) if self._serialize else data
 
     def request(self, method=None, path=None, **request):
+        """Internal method which calls Syncano API and returns serialized data."""
```

Comment:
It's actually a restrictive clause. There should be no comma, even when using which.